android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         mavenLocal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.1'
+        classpath 'com.android.tools.build:gradle:4.2.0-beta04'
```

Comment:
I think it's best to stay with stable versions and not beta unless it is necessary.